### PR TITLE
New event structure yaml append fron matter.

### PR DESCRIPTION
**Pull request by amed**
@jhilden 
Please check this, I separated the new structure in to two pieces, 
this error caused because it is rendering the old pages and looking for new front maters data (that are new appended data to remote event)

The pull request (New event structure files https://github.com/webmontag/webmontag-koeln.de-reloaded/pull/59) is the new data structure.

### CODE REVIEW

**Comment by jhilden** [comment]
Deploy preview failed.

Built with commit 6f66200735b618cbf3336c23c76a2d2157fb5e56

https://app.netlify.com/sites/webmontag/deploys/5824a3efc4d9cc42a08614f1

File: lib/events-scraper/event.rb [hunk 1]
```diff
@@ -69,4 +72,51 @@ def remote_document
       Nokogiri::HTML(open(uri), nil,"UTF-8")
     end
   end
+
+  def front_matter
```
Comment (jhilden) [inline lib/events-scraper/event.rb:76]: split up into 2 methods:
- `metadata`
- `front_matter`

File: lib/events-scraper/event.rb [hunk 2]
```diff
@@ -69,4 +72,51 @@ def remote_document
...
+    #   	url
+    #
+
+    hash = Hash.new
+
+    hash = {
+      name: "Webmontag",
+      title: name,
+      url: "#{EVENTS_URL}#{@date_string}",
+      image: EVENTS_IMAGE, # it have to be set as a permanent link.
```
Comment (jhilden) [inline lib/events-scraper/event.rb:104]: the constant `EVENTS_IMAGE` is only used once.  maybe this is prematuire optimization?

File: lib/events-scraper/event.rb [hunk 3]
```diff
@@ -69,4 +72,51 @@ def remote_document
...
+      description:  "Web Monday is an informal, non-commercial, and completely community-driven event that aims to connect the people who are shaping the future of the internet. Inspired by the culture of Silicon Valley, it started out in Cologne, Germany in late 2005 in an effort to help spread those sunny California vibes.",
+      startDate: "#{@date_string}T19:00",
+      endDate: "#{@date_string}T21:00",
+      performer: { name: " " },
+      location: {
+        name: "Die Wohngemeinschaft - Theater",
+        address: {
+          streetAddress: "Richard-Wagner-Straße 39", postalCode: "50674", addressLocality: "Cologne", addressCountry: { name: "Germany"  }}},
```
Comment (jhilden) [inline lib/events-scraper/event.rb:112]: write closing braces so you can see where location ends

File: lib/events-scraper/event.rb [hunk 4]
```diff
@@ -69,4 +72,51 @@ def remote_document
...
+    hash = {
+      name: "Webmontag",
+      title: name,
+      url: "#{EVENTS_URL}#{@date_string}",
+      image: EVENTS_IMAGE, # it have to be set as a permanent link.
+      description:  "Web Monday is an informal, non-commercial, and completely community-driven event that aims to connect the people who are shaping the future of the internet. Inspired by the culture of Silicon Valley, it started out in Cologne, Germany in late 2005 in an effort to help spread those sunny California vibes.",
+      startDate: "#{@date_string}T19:00",
+      endDate: "#{@date_string}T21:00",
+      performer: { name: " " },
+      location: {
```
Comment (jhilden) [inline lib/events-scraper/event.rb:109]: `if (Date.parse(@date_string).year >= 2016`

File: source/assets/stylesheets/events.css [hunk 5]
```diff
@@ -1,4 +1,20 @@
-#content {margin: 2% 5%;}
+.event-ticket{
```
Comment (jhilden) [inline source/assets/stylesheets/events.css:1]: ticket??

Comment (jhilden) [reply source/assets/stylesheets/events.css:1]: maybe "event-hero"?

File: source/assets/stylesheets/events.css [hunk 6]
```diff
@@ -1,4 +1,20 @@
...
+  border-bottom: 1px solid #ddd;
+}
+.event-ticket .description{
+  padding: 10px 0 0 20px;
+}
+.event-ticket .further-info{
+  padding: 10px 0 0 20px;
+}
+.event-ticket .event-image{
+
```
Comment (jhilden) [inline source/assets/stylesheets/events.css:13]: empty :(

**Review by jhilden — APPROVED** [review]

File: lib/events-scraper/event.rb (hunk 4 above)
Comment (amed) [reply lib/events-scraper/event.rb:109]: We will figure it out later.

**Pull request merged**